src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -880,7 +880,9 @@ void S6Exporter::ExportResearchedSceneryItems()
 
     for (uint16_t sceneryEntryIndex = 0; sceneryEntryIndex < RCT2_MAX_RESEARCHED_SCENERY_ITEMS; sceneryEntryIndex++)
```

Comment:
There's still similar define around: `MAX_RESEARCHED_SCENERY_ITEMS`, not used any more. Can you delete it now?